meilisearch-http/src/routes/swap_indexes.rs

```diff
@@ -16,11 +16,10 @@ use crate::routes::tasks::TaskView;
 pub fn configure(cfg: &mut web::ServiceConfig) {
     cfg.service(web::resource("").route(web::post().to(SeqHandler(swap_indexes))));
 }
-
```

Comment:
Why? ahahaha

Reply:
oh god I don't know how I always end up messing with the newlines 😭